ElastosDIDSDK/Backend/IDTransactionInfo.swift

```diff
@@ -32,75 +33,75 @@ public class IDTransactionInfo: DIDTransaction {
     ///   - transactionId: The transaction id string.
     ///   - timestamp: The time stamp
     ///   - request: The IDChainRequest content.
-    public init(_ transactionId: String, _ timestamp: Date, _ request: IDChainRequest) {
+    @objc public init(_ transactionId: String, _ timestamp: Date, _ request: IDChainRequest) {
```

Comment:
Can be formatted as below:
@objc
public init(_ transactionId: String, _ timestamp: Date, _ request: IDChainRequest) {